tests/jballerina-debugger-integration-test/src/main/java/org/ballerinalang/debugger/test/utils/DebugHitListener.java

```diff
@@ -45,7 +45,7 @@
     private boolean debugHitFound;
 
     public DebugHitListener(TestDAPClientConnector connector) {
-        this.connector = connector;
+        DebugHitListener.connector = connector;
```

Comment:
`connector` is a static variable. Therefore I'm accessing here with static `DebugHitListener.connector` via class `DebugHitListener` reference instead of accessing via instance reference. Even the IntelliJ IDEA suggests the same.

Reply:
Changed to `this.connector = connector;` after restructuring the test utils.